packages/amplify-e2e-tests/src/__tests__/rds-import-vpc.test.ts

```diff
@@ -19,6 +19,13 @@ import gql from 'graphql-tag';
 import path from 'path';
 import { print } from 'graphql';
 
+const CDK_FUNCTION_TYPE = 'AWS::Lambda::Function';
+const CDK_SUBSCRIPTION_TYPE = 'AWS::SNS::Subscription';
+const APPSYNC_DATA_SOURCE_TYPE = 'AWS::AppSync::DataSource';
+
+const SNS_TOPIC_REGION = 'us-east-1';
+const SNS_TOPIC_ARN = 'arn:aws:sns:us-east-1:956468067974:AmplifyRDSLayerNotification';
```

Comment:
which account does this correspond to? Can we define this as a constant with an intuitive name?

Reply:
All RDS service resources point to beta account now. Not sure about moving the account number to a different constant will add anything to the clarity. I'm going to have a follow up PR to change all these URLs to point to production account. I will add necessary details in comments for more clarity.

Reply:
What is the difference between using `beta` vs `prod`. Generally we want to test our changes on `beta` before pushing to `prod`, is there a documented way to achieve this?

Reply:
The ARN is different for beta and prod. We can have both in the comments. If required, we can toggle the URL and test it against beta. 

We can also add a flag '--use-rds-beta'. If passed, we can deploy with beta layer and SNS notification. But that can be added iteratively, not necessarily on this PR.